### PR TITLE
Add cloud-native AEM Events documentation and align other AEM docs accordingly

### PR DESCRIPTION
**Pull request by shikhartanwar**
## Description

Add cloud-native AEM Events documentation and align other AEM docs accordingly

## Motivation and Context
- Added documentation for cloud-native AEM Events (directly referenced the experience-manager-apis repo to avoid duplication).
- Refactored side nav for AEM docs
- Added emphasis on using cloud-native AEM Events
- Added deprecation notice wherever applicable
- Removed docs for AEM 6.4

## How Has This Been Tested?

`npm run dev`

## Screenshots (if appropriate):

| S. No. | Screenshot |
|--------|--------|
| 1 | aem-events-landing-page |
| 2 | aem-cloud-service-events-landing-page |
| 3 | aem-add-on-module-events-landing-page
| 4 | aem-add-on-module-for-aem-cs-events-landing-page |

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rperelma** [comment]
@shikhartanwar thank you!!!
Would it make sense to reference https://experienceleague.adobe.com/docs/experience-manager-learn/cloud-service/aem-eventing/overview.html ?
cc: @peterklassen

**Comment by shikhartanwar** [comment]
> @shikhartanwar thank you!!! Would it make sense to reference https://experienceleague.adobe.com/docs/experience-manager-learn/cloud-service/aem-eventing/overview.html ? cc: @peterklassen

@rperelma That doc is already embedded from AEM's doc repo (see 2nd screenshot). So, basically the same page lives under both domains.

**Comment by shikhartanwar** [comment]
> @shikhartanwar thank you!!! Would it make sense to reference https://experienceleague.adobe.com/docs/experience-manager-learn/cloud-service/aem-eventing/overview.html ? cc: @peterklassen

Oh this is another page. Yes we can try and include this one.

**Comment by peterklassen** [comment]
image
 we cant recommend this to non AEM -CS customers as this is available only on CS. And we still route new customers like Coke to it. I thought we had discussed to deprecate and recommend CS Events for AEM CS customers. Stating that we don’t maintain customers production code anymore - not sure that’s the best option…

**Comment by peterklassen** [comment]
image
Another catch: In the navigation bar, you have Cloud-Native Events below AEM Events and the AEM Cloud Events again under AEM Events powered by AEM add-on module.. sort of confusing structure..

**Comment by peterklassen** [comment]
image
CS installation guide on that page - misleading

**Comment by peterklassen** [comment]
image
I feel like this should not be called AEM on-Prem Events, it should be "AEM Events on AEM On-Prem" and "AEM Events Add-On Module on AEM On-Prem"- just want to avoid confusion reg the hybrid mode with classic events on AEM CS and calling it Cloud Events

File: src/pages/guides/using/aem/cloud-native/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: AEM Events as a Cloud Service
+description: Introduction to the all-new AEM Events in AEM as a Cloud Service
```
Comment (kgera) [inline src/pages/guides/using/aem/cloud-native/index.md:3]: @peterklassen  Please can you review Wording 'AEM Events in AEM as a Cloud Service' it lokks really long and not very developer friendly. cc @shikhartanwar

File: src/pages/guides/using/aem/index.md [hunk 2]
```diff
@@ -4,28 +4,5 @@ title: AEM Events
...
-
-## Use Adobe IO Events
-Once the previous steps are completed, a new AEM provider should appear in the Events Providers list in your Adobe Developer Console.
-Please refer to the Adobe Developer Console documentation on how to [add Events to a project](/developer-console/docs/guides/services/services-add-event/) if you are new to this.
-
-![Adobe Developer Console showing an AEM Events Provider](../../img/add_skyline_event_provider.png "Adobe Developer Console showing an AEM Events Provider")
-
-you are ready to register a new [webhook](../../index.md)
-or to start pulling events from this new AEM source using the [journaling](../../journaling_intro.md).
+1. **[Cloud-Native AEM Events in AEM as a Cloud Service](./cloud-native/index.md)**
```
Comment (kgera) [inline src/pages/guides/using/aem/index.md:7]: @peterklassen  to review naming convection, additionally what does AEM PMM team recommends.

**Review by kgera — COMMENTED** [review]
@shikhartanwar  The naming convection for AEM Eventing looks very long. I commented @peterklassen  to review and consult AEM Marketing team.

**Comment by kgera** [comment]
> image we cant recommend this to non AEM -CS customers as this is available only on CS. And we still route new customers like Coke to it. I thought we had discussed to deprecate and recommend CS Events for AEM CS customers. Stating that we don’t maintain customers production code anymore - not sure that’s the best option…



> image we cant recommend this to non AEM -CS customers as this is available only on CS. And we still route new customers like Coke to it. I thought we had discussed to deprecate and recommend CS Events for AEM CS customers. Stating that we don’t maintain customers production code anymore - not sure that’s the best option…

I would suggest we change this as following: 

"We highly recommend AEM CS customers to use ...."

**Comment by shikhartanwar** [comment]
> image we cant recommend this to non AEM -CS customers as this is available only on CS. And we still route new customers like Coke to it. I thought we had discussed to deprecate and recommend CS Events for AEM CS customers. Stating that we don’t maintain customers production code anymore - not sure that’s the best option…

@peterklassen: Based on the image alone, I'm unable to understand the full context. Are you suggesting that we remove this notice from the file [aem_on_premise_install.md](https://github.com/AdobeDocs/adobe-io-events/blob/2a834b33b2118fe29e0f96a1b7ff671b6be1b4a7/src/pages/guides/using/aem/aem-addon-module/aem_on_premise_install.md), or are you proposing a change in the recommendation itself?

**Comment by shikhartanwar** [comment]
> image Another catch: In the navigation bar, you have Cloud-Native Events below AEM Events and the AEM Cloud Events again under AEM Events powered by AEM add-on module.. sort of confusing structure..

The structure I had in mind was to differentiate between the cloud-native offering and the legacy offering. However, I'm open to suggestions here.

**Comment by shikhartanwar** [comment]
@peterklassen @kgera With all the naming suggestions, I am just thinking doing a live review would be much much better to get us to a better state.

**Comment by shikhartanwar** [comment]
@kgera @peterklassen I have incorporated feedback from our last discussion. Please do a final review.

**Pull request merged**